assets/data-port/style.scss

```diff
@@ -115,6 +115,10 @@ $alert-red: #d94f4f;
 			border-bottom: 1px solid #eee;
 			margin: 0;
 			padding: 24px 24px 0;
+
+			a {
+				font-size: 16px;
```

Comment:
Maybe we could use `font-size: inherit` here. So when we change the `p` font-size it's automatically changed too.

Reply:
https://github.com/Automattic/sensei/pull/3452/commits/fc4d7444babdb86862fce503e8e9fe18c2a04aaa